tools/targets/NotSupported.targets

```
@@ -48,11 +49,12 @@
       <GenAPIPath Condition="'$(MSBuildRuntimeType)' == 'core'"> "$(DotNetCmd) $(ToolsArtifactsDir)netcoreapp2.1\Microsoft.DotNet.GenAPI.dll"</GenAPIPath>
       <GenAPIPath Condition="'$(MSBuildRuntimeType)' != 'core'"> "$(ToolsArtifactsDir)net472\Microsoft.DotNet.GenAPI.exe"</GenAPIPath>
     </PropertyGroup>
+    <message Condition="'$(BuildTools)' != 'true'" Text="Skip building tools." />
```

just to keep it consistentwith other tags can you change this to:
```suggestion
    <Message Condition="'$(BuildTools)' != 'true'" Text="Skip building tools." />
```